fvm/errors.go

```diff
@@ -198,6 +200,25 @@ func (e *InvalidProposalKeyMissingSignatureError) Code() uint32 {
 	return errCodeInvalidProposalKeyMissingSignature
 }
 
+// MaxEventSizeLimitExceededError indicates that the transaction has produced events with size more than limit.
+type MaxEventSizeLimitExceededError struct {
```

Comment:
"Max" and "Limit" are redundant, i would say `EventSizeLimitExceededError` or just `EventLimitExceededError`

Reply:
Applied at 5167714